google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/vcs/GcpCheckoutProvider.java

```diff
@@ -186,8 +190,24 @@ private static boolean doClone(@NotNull Project project, @NotNull ProgressIndica
       return true;
     }
     VcsNotifier.getInstance(project)
-        .notifyError(GctBundle.message("clonefromgcp.failed"), result.getErrorOutputAsHtmlString());
+        .notifyError(GctBundle.message("clonefromgcp.failed"), result.getErrorOutputAsHtmlString()
+            + "<br>" + resultWithHtmlUrl(result.getOutput()), new UrlOpeningListener(true));
     return false;
   }
 
+  /**
+   * Searches for URLs inside a string list and adds a HTML "a" element, so they're openable from
+   * the notification.
+   */
+  private static String resultWithHtmlUrl(List<String> output) {
```

Comment:
@etanshaul PTAL

I decided to try for URLs using startsWith instead of new URL(), so I wouldn't have to handle exceptions.